src/js/ui/loginModal.ts

```diff
@@ -48,9 +48,16 @@ export default {
           m('button.fat', i18n('signIn'))
         ]),
         m('div.signup', [
+          i18n('newToLichess') + ' ',
           m('a', {
             oncreate: helper.ontap(signupModal.open)
-          }, [i18n('newToLichess'), ' ', i18n('signUp')])
+          }, [i18n('signUp')])
+        ]),
+        m('div.reset', [
+          i18n('forgotPassword') + ' ',
+          m('a', {
```

Comment:
It's better using cordova inappbrowser plugin (https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/) rather than a link:

```
window.open('https://en.lichess.org/password/reset')
```

better: you can open the localized page using the detected lang in app:

```
window.open(`https://${settings.general.lang()}.lichess.org/password/reset`)
```